src/main/resources/db/migration/V057__Add_document_order_column_to_scannable_items.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+  ADD COLUMN documentOrder INTEGER DEFAULT 0;
```

Comment:
Best to add not null contraint.

Reply:
how we can sort out existing ones?

Reply:
I thought that's why you added the default. Ultimately we shouldn't even have default value for this column, as we'll be always setting it from code.